prowler/lib/check/models.py

```diff
@@ -113,9 +113,9 @@ class CheckMetadata(BaseModel):
     ResourceType: str
     Description: str
     Risk: str
-    RelatedUrl: Optional[str] = None
+    RelatedUrl: str = Field(default_factory=str)
```

Comment:
Do you need this? As far as I know a `str` in Python is immutable so you should not need the default factory. Maybe it's a good practice to keep.